client/src/helpers/APICalls/uploadImage.ts

```diff
@@ -0,0 +1,20 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+interface Props {
+  file: File;
+}
+
+export async function uploadImage({ file }: Props) {
+  const data = new FormData();
+  data.append('image', file);
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    credentials: 'include',
+    body: data,
+  };
+  return await fetch(`/profile/upload-image`, fetchOptions)
+    .then((res) => console.log(res.json()))
```

Comment:
Let's get rid of the console statement here 👍🏼

Reply:
OK